frontend/src/lib/components/TaxonomicPopover/TaxonomicPopover.tsx

```diff
@@ -59,8 +59,8 @@ export function TaxonomicPopover<ValueType extends TaxonomicFilterValue = Taxono
         <span className={placeholderClass ?? 'text-muted'}>{placeholder}</span>
     )
     buttonPropsFinal.onClick = () => setVisible(!visible)
-    buttonPropsFinal.status = 'stealth'
-    buttonPropsFinal.type = 'secondary'
+    buttonPropsFinal.status ||= 'stealth'
+    buttonPropsFinal.type ||= 'secondary'
```

Comment:
Whaaaaat is this?

Reply:
[ES2021](https://h3manth.com/ES2021/), but it seems this is breaking the storybook build

Reply:
Darnit. I don't have time to debug that today 😬